var/spack/repos/builtin/packages/argobots/package.py

```diff
@@ -41,4 +42,7 @@ def configure_args(self):
         else:
             args.append('--disable-valgrind')
 
+        if '+debug' in self.spec:
+            args.append('--enable=debug=yes')
```

Comment:
Shouldn't it be `--enable-debug`, not `--enable=debug`? Also, can you explicitly disable this in an else clause?